### DISCUSSION TITLE
Customizing the Hits component

### DISCUSSION THREAD

**Discussion by theunreal** [algolia/instantsearch#5430] [Q&A]
I'm trying to do something basic, and after digging through the docs I couldn't find a proper solution.

I want to transform the Hits component to a responsive list, which means I need to control the rendered HTML to implement grid CSS and not use the default classes. How I can do that?

**Comment by Haroenv** [comment]
You can pass custom css classes using the `cssClasses` option. Assuming you're using InstantSearch.js, the syntax would be:

```js
hits({
  cssClasses: {
    list: 'd-grid something',
    item: 'something something-else'
  }
});
```

hope that helps!

If you need to customise the html for some reason even more than that, you can use `connectHits` to render completely on your own.
